docs/Tutorials/BeginnersTutorial.md

```diff
@@ -221,8 +219,12 @@ Accepting connection(s): 92bbb69f2af2:11111
 
 This means it's waiting for you to connect some external Paraview session (the
 "client") to the server. Now, ***outside*** the container, start a session of
-Paraview 5.10.1. (Again, you must use this version otherwise it won't work
-properly.) Go to `File > Connect`. Click `Add Server`. Name it whatever you
+Paraview. 
+
+\note To figure out the version of ParaView to use, run `mpirun -n 1 pvserver --version`, then download ***exactly the same*** version of Paraview. At the time when writing this tutorial, we are using ParaView 5.11.1  
```

Comment:
I like the placement of the note here, but I'd keep the original wording all the way through "won't work properly.)" in the first paragraph. Then have your `\note` but clarify that that command you wrote is only if you are using a cluster. Also, since we sometimes have different versions of Paraview on different clusters, we should avoid putting a version in this `\note`.

Then the next paragraph starts exactly like you have it with "Go to..."